src/session.ts

```diff
@@ -375,6 +374,71 @@ export async function getTableHtml(webview: Webview, file: string): Promise<stri
         padding: 0;
         overflow: auto;
     }
+
+    /* Styling for wrapper and header */
+
+    [class*="vscode"] div.ag-root-wrapper {
+        background-color: var(--vscode-editor-background);
+    }
+
+    [class*="vscode"] div.ag-header {
+        background-color: var(--vscode-sideBar-background);
+    }
+
+    [class*="vscode"] div.ag-header-cell[aria-sort="ascending"], div.ag-header-cell[aria-sort="descending"] {
+        color: var(--vscode-textLink-activeForeground);
+    }
+
+    /* Styling for rows and cells */
+
+    [class*="vscode"] .ag-row-selected {
```

Comment:
Is `.ag-row-hover` missing? The theme has a hover item.

Reply:
Do you think sharing the `.ag-row-selected` styling would be okay? Or should it be a unique colour?

Reply:
`.ag-row` should appear not highlighted, `.ag.row-hover` slightly highlighted, `.ag-row-selected` more highlighted. So I think it is different from `.ag-row-selected` or otherwise user cannot distinguish hovered rows and selected rows.

Reply:
I have added a `.ag-row-hover`.